.github/workflows/indy_image_build.yaml

```diff
@@ -0,0 +1,132 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Build indy os docker images triggered by :
+#  - push on develop branch for latest tags 
+#  - create tag vx.y.z for release tags
+#  - Can be triggered manually
+
+name: Build indy os docker images
```

Comment:
Rename to "Build Indy docker images for Bevel"

Reply:
changed.